### PR TITLE
Grace: new package

### PR DESCRIPTION
**Pull request by RemiLacroix-IDRIS**

### CODE REVIEW

File: var/spack/repos/builtin/packages/grace/package.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Grace(AutotoolsPackage):
+    """Grace is a WYSIWYG 2D plotting tool for the X Window System and M*tif."""
+
+    homepage = "http://plasma-gate.weizmann.ac.il/Grace"
+    url      = "ftp://ftp.fu-berlin.de/unix/graphics/grace/src/grace5/grace-5.1.25.tar.gz"
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/grace/package.py:15]: Curious.  Why are you using the mirror from Germany instead of the g FTP site?

Comment (RemiLacroix-IDRIS) [reply var/spack/repos/builtin/packages/grace/package.py:15]: I haven't been able to access `ftp://plasma-gate.weizmann.ac.il/pub/grace/` in the last months. Does it work for you?

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/grace/package.py:15]: Fair enough.

Comment (tldahlgren) [reply var/spack/repos/builtin/packages/grace/package.py:15]: Can you add a brief comment above line 13 to explain use of the mirror?

Comment (RemiLacroix-IDRIS) [reply var/spack/repos/builtin/packages/grace/package.py:15]: Done.

**Review by tldahlgren — APPROVED** [review]
LGTM

**Pull request merged**